scripts/packaging/installrequirements.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
+
+GOVERSION="${GOVERSION:-1.20.4}"
```

Comment:
We can use a more recent version of go, like 1.21.3, or 1.20.10

Reply:
Switched to `actions/setup-go@v4` using go version '1.21.x'